en/administration/secure-platform.md

```diff
@@ -37,6 +37,133 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
+directories?
+
+The standard access policy based on the user, group, and other permissions, known as Discretionary Access Control
+(DAC), does not enable system administrators to create comprehensive and fine-grained security policies, such as
+restricting specific applications to only viewing log files, while allowing other applications to append new data to
+the log files.
+
+SELinux implements Mandatory Access Control (MAC). Every process and system resource has a special security label
+called an SELinux context. A SELinux context, sometimes referred to as an SELinux label, is an identifier which
+abstracts away the system-level details and focuses on the security properties of the entity. Not only does this provides
+a consistent way of referencing objects in the SELinux policy, but it also removes any ambiguity that can be found in
+other identification methods. For example, a file can have multiple valid path names on a system that makes use of bind
+mounts.
+
+The SELinux policy uses these contexts in a series of rules which define how processes can interact with each other and
+the various system resources. By default, the policy does not allow any interaction unless a rule explicitly grants access.
+
+For more information about SELinux please see [Red Hat documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_selinux/getting-started-with-selinux_using-selinux)
+
+### Activer SELinux en mode permissif
+
+By default, SELinux is disabled during Centreon installation process. To enable SELinux in permissive mode, you need to
+modify the `/etc/selinux/config` file as:
+
+```shell
+# This file controls the state of SELinux on the system.
+# SELINUX= can take one of these three values:
+#     enforcing - SELinux security policy is enforced.
+#     permissive - SELinux prints warnings instead of enforcing.
+#     disabled - No SELinux policy is loaded.
+SELINUX=permissive
+# SELINUXTYPE= can take one of three two values:
+#     targeted - Targeted processes are protected,
+#     minimum - Modification of targeted policy. Only selected processes are protected.
+#     mls - Multi Level Security protection.
+SELINUXTYPE=targeted
+```
+
+Then reboot your server:
+```shell
+shutdown -r now
+```
+
+### Install Centreon SELinux packages
+
+Depending on the type of server, install the packages with the following command:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Central / Remote Server-->
+   ```shell
+   yum install centreon-common-selinux \
+   centreon-web-selinux \
+   centreon-broker-selinux \
+   centreon-engine-selinux \
+   centreon-gorgoned-selinux \
+   centreon-plugins-selinux
+   ```
+<!--Poller-->
+   ```shell
+   yum install centreon-common-selinux \
+   centreon-broker-selinux \
+   centreon-engine-selinux \
+   centreon-gorgoned-selinux \
+   centreon-plugins-selinux
+   ```
+<!--Map server-->
+   ```shell
+   yum install centreon-map-selinux
```

Comment:
Isn't there a dependency between map and centreon-common?

Reply:
I didn't think so